### PR TITLE
Export sandboxes from dashboard

### PR DESCRIPTION
**Pull request by gabeeden**
**What kind of change does this PR introduce?**
Feature

**What is the current behavior?**
#1844

**What is the new behavior?**
Allows exporting a single Sandbox and multiple Sandboxes from the dashboard

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table 

The getSandbox function is copied from here (with track removed):
https://github.com/codesandbox/codesandbox-client/blob/5f09dec3a59fdec5b4f63c48feed6b050047c0f3/packages/homepage/src/screens/explore/SandboxModal.js#L46

Not sure if this should be duplicated here?

### CODE REVIEW

File: packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx [hunk 1]
```diff
@@ -217,6 +218,13 @@ class SandboxItem extends React.PureComponent<Props, State> {
             },
             color: theme.red.darken(0.2)(),
           },
+          {
```
Comment (CompuIves) [inline packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx]: Could you put this one above the  `Move to Trash` with a separator? So it  would be:

```
Privacy controls
---
Export
---
Move to trash
```

File: packages/app/src/app/pages/Dashboard/Content/SandboxGrid/index.js [hunk 2]
```diff
@@ -116,6 +121,39 @@ class SandboxGrid extends React.Component<*, State> {
     setSandboxesPrivacy(this.props.store.dashboard.selectedSandboxes, privacy);
   };
 
+  getSandbox = async sandboxId => {
```
Comment (CompuIves) [inline packages/app/src/app/pages/Dashboard/Content/SandboxGrid/index.js]: This is fine!

**Comment by gabeeden** [comment]
Yeah done!

**Comment by CompuIves** [comment]
Perfect!! Thanks a bunch for this contribution!!

@allcontributors[bot] Please add @gabeeden for code

**Comment by CompuIves** [comment]
Hmmm

@allcontributors[bot] Please add @gabeeden for code

**Pull request merged**